brian2/devices/cpp_standalone/device.py

```diff
@@ -480,9 +480,28 @@ def variableview_get_with_expression(self, variableview, code,
                                   'variables with string expressions in '
                                   'standalone scripts.')
 
-    def code_object_class(self, codeobj_class=None):
-        # Ignore the requested codeobj_class
-        return CPPStandaloneCodeObject
+    def code_object_class(self, codeobj_class=None, pref=None):
```

Comment:
This still says `pref` instead of `fallback_pref` -- standalone simulations therefore fail.